src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralService.kt

```diff
@@ -161,7 +161,7 @@ class ReferralService(
       referral.id,
       referral.serviceUserCRN,
       user,
-      OffsetDateTime.now(), // fixme: this should be the timestamp which we store when additionalRiskInformation is set
+      referral.additionalRiskInformationUpdatedAt,
       riskInformation,
     )
 
```

Comment:
should we clear the timestamp below too??

Reply:
would be good to retain it until we have a way to retrieve it from ARN

Reply:
👍🏼